doc/examples/segmentation/plot_regionprops.py

```diff
@@ -47,3 +58,12 @@
 
 ax.axis((0, 600, 600, 0))
 plt.show()
+
+props = regionprops_table(label_img, properties=('centroid',
```

Comment:
We could perhaps say here that `regionprops_table` actually computes the properties, while `regionprops` only does lazy execution.

Reply:
Why not; done in 38ddd671d81db0eb69f28351094ed4ea995511dc.